app/views/users/_form.html.erb

```diff
@@ -88,7 +88,8 @@
     <% end %>
 
     <div class='tab-pane' id='roles'>
-      <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
+      <% caption = @user.inherited_admin? ? _('Admin rights are inherited from UserGroup') : '' %>
```

Comment:
```suggestion
      <% caption = @user.inherited_admin? ? "#{_('Admin rights are currently inherited from')} #{_('Usergroup')}" : '' %>
```
If you refer to a model `Usergroup`, it is a small g and we can reuse the translation of the model if we split the translation to pieces. Or you could just use `user group` as a concept, not the model, both is fine with me :)
I would  like to see `currently` or another timely word to refer to the temporar fashion of the inheritance